### ISSUE TITLE
"returns false if there is a connection timeout" test fails

### ISSUE THREAD

**Issue by delfigamer** [jfischoff/port-utils#2]
```
Failures:

  test/Tests/Network/Socket/WaitSpec.hs:95:7:
  1) Tests.Network.Socket.Wait.connectAction returns false if there is a connection timeout
       expected: False
        but got: True
```

https://github.com/jfischoff/port-utils/blob/7e512ae325c78915b49603fca683792f47e71137/test/Tests/Network/Socket/WaitSpec.hs#L93-L96

Going to the specified address: http://93.184.216.34/, reveals a functioning web server that immediately responds with a proper 404.

**Comment by jfischoff** [comment]
Makes sense. Not sure why I choose that IP address. Clearly need something better.

**Comment by dustin** [comment]
This is causing a dependency failure via a nix build pipeline in CI.  Can a release disable this test?  It's a quite misleading test.

**Comment by robinp** [comment]
Message should say "on connection failure" probably, and safer to choose localhost IP with a bogus port (less likely to upset anyone).
